db/memtable_list.cc

```diff
@@ -578,17 +578,17 @@ Status MemTableList::TryInstallMemtableFlushResults(
         }
 
         edit_list.push_back(&m->edit_);
-        memtables_to_flush.push_back(m);
         std::unique_ptr<FlushJobInfo> info = m->ReleaseFlushJobInfo();
         if (info != nullptr) {
           committed_flush_jobs_info->push_back(std::move(info));
         }
       }
-      batch_count++;
+      memtables_to_flush.push_back(m);
     }
 
+    size_t num_mem_to_flush = memtables_to_flush.size();
     // TODO(myabandeh): Not sure how batch_count could be 0 here.
-    if (batch_count > 0) {
```

Comment:
Renamed for better understanding.

Reply:
It seems the variable is not named correctly since it was introduced: https://github.com/facebook/rocksdb/commit/5aaef91d4ab3ba2ca6f2566647049c2a2b55281e#diff-398125fa2b06616bca3e9aaeeacb38d0f251321cc15722e4b636d20d1dfb1d6a. The way it's used seem to be number of memtables flushed.